collector/postgres/collector.yml

```diff
@@ -0,0 +1,31 @@
+receivers:
+  postgresql:
+    endpoint: postgres:5432
+    username: postgres
+    password: postgres
+    databases:
+      - postgres
+    collection_interval: 5s
+    tls:
+      insecure: true
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    loglevel: debug
+  otlp:
+    endpoint: ingest.staging.lightstep.com:443
```

Comment:
Let's change this to `ingest.lightstep.com:443` instead (as actual final users will see this).

Reply:
Done